lib/l10n/app_en.arb

```diff
@@ -601,5 +601,7 @@
   "topSongs": "Top Songs",
   "@topSongs": {},
   "playCountInline": " · {playCount} plays",
-  "@playCountInline": {}
+  "@playCountInline": {},
+  "couldNotLoadAlbum": "Couldn't load album",
+  "@couldNotLoadAlbum": {}
```

Comment:
Maybe use a select here instead, to support different types as well? Like in line 588